### DISCUSSION TITLE
Skip sync only for download, but not upload

### DISCUSSION THREAD

**Discussion by koh-osug** [abraunegg/onedrive#2402] [Q&A] [answered]
I want to upload all files, but only skip some files which are too large on my travel computer, e.g. upload all images + raw files, but not download them. Is this possible?

**Comment by abraunegg** [comment, marked as answer]
@koh-osug 
> I want to upload all files, but only skip some files which are too large on my travel computer, e.g. upload all images + raw files, but not download them. Is this possible?

Please read the documentation, this is explained in usage.md

**Comment by koh-osug** [comment]
I have just found `skip_file` which is not working for this. Is there later a `skip_download_file` which I have missed?

**Reply by abraunegg** [reply]
> I have just found skip_file which is not working for this. Is there later a skip_download_file which I have missed?

`skip_file` works for upload and download.

What application version are you using? What is the output of of `onedrive --version` ?

**Comment by koh-osug** [comment]
onedrive v2.4.23-2-g3acb6c1

skip_file seems to work fine, but it skips e.g. all `orf` files for the upload and download. But I want to upload orf files. And not download them for other folders. I need:

skip_download_file="*.orf"

**Reply by abraunegg** [reply]
> skip_file seems to work fine, but it skips e.g. all orf files for the upload and download. But I want to upload orf files. And not download them for other folders. I need:

You will need to run 2 client configurations then:

* configuration one 'your upload directive': `onedrive --confdir=~/.config/upload-config` --synchronize --upload-only --verbose <any other options like --no-remote-delete>
* configuration two 'your download directive': `onedrive --confdir=~/.config/download-config` --synchronize --download-only --verbose <any other options required>

In each of the config folders, download a copy of the configuration file and store in the different configuration folders.

**Reply by koh-osug** [reply]
This seems to fit my needs then. Thanks.
